elfloader-tool/src/plat/apq8064/platform_init.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+#include <autoconf.h>
+#include <mode/arm_generic_timer.h>
+
+/* Reset the virtual offset for the platform timer to 0 */
+void platform_init(void)
+{
+    reset_cntvoff();
```

Comment:
If someone wanted to deduplicate this code in the future, then adding the `/timer` device to the `seL4,elfloader-devices` property in the device trees for the relevant platforms and adding a generic-timer ELFLOADER_DRIVER that resets the cntvoff would probably be the way to do it.

Reply:
It's not clear what the best way to modify this might be given that the timer needs to be initialized once per for from the given core. Maybe introducing a timer interface to reset the timer on each core?